opentracing/src/main/java/opentracing/Span.java

```diff
@@ -25,16 +25,7 @@
   /**
    * Suitable for serializing over the wire, etc.
    */
-  TraceContext traceContext();
-
-  /**
-   * Denotes the beginning of a subordinate unit of work.
-   *
-   * @param operationName name of the operation represened by the new span from the perspective of
-   * the current service.
-   * @return a new child Span in "started" state.
-   */
-  Span startChild(String operationName);
+  SpanContext childContext();
```

Comment:
> Is the returned childContext immutable in this design?

No.
For example, using the two scenarios in https://github.com/opentracing/opentracing-java/pull/7#discussion_r50500237

1) 

    SpanContext spanContext = traceContext.newRoot();  // or traceContext.spanContext();
    spanContext.setAttribute("foo", "bar");
    Tracer.startTrace("new-span-name", spanContext);

2) 

    childContext = currentSpan.childContext();
    childContext.setAttribute("foo", "bar");
    Tracer.joinTrace("new-span-name", childContext);

I do think once we settle on on SpanContexts are used, we can improve how we define Tracer.